src/Components/Home/Home.css

```diff
@@ -19,4 +19,10 @@ span.p-chip-text{
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+}
+
+.user-list{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
We use PrimeReact, this can be achieved without custom code https://www.primefaces.org/primereact/showcase/#/flexbox

Reply:
I will take a look. 👍